recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
     - preshed >=3.0.2,<3.1.0
     - murmurhash >=0.28.0,<1.1.0
     - cython-blis >=0.4.0,<0.8.0
+    - pydantic >=1.7.4,!=1.8,!=1.8.1,<1.9.0  # [arm64]
```

Comment:
Is this dependency just required on arm64?

Reply:
Yeah, there was an error only on `arm64` in host env